### ISSUE TITLE
email_not_verified error

### ISSUE THREAD

**Issue by k5jat** [LemmyNet/lemmy#3079]
- [x] Did you check to see if this issue already exists?
- [x] Is this only a single bug? Do not put multiple bugs in one issue.
- [n] Is this a question or discussion? Don't use this, use https://lemmy.ml/c/lemmy_support .
- [n] Is this a UI / front end issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo.

### Issue Summary

I keep getting this when I try to login to my lemmy site. This is after I turned on "Require email verification." It allows normal login after the option is turned off. It seem the initial admin email during setup is not marked as verified.

P.S. I tried searching for this bug and didn't see anything that resembles this, so forgive me if it's a dupe.

### Steps to Reproduce

1. Setup lemmy as per normal
2. Turn on "Require email verification" in settings
3. Login elsewhere (my phone) and the log spits out the error below.

### Technical details

- Please post your log: `sudo docker-compose logs > lemmy_log.out`.
```
lemmy_1     | 2023-06-13T22:51:42.452680Z ERROR HTTP request{http.method=GET http.scheme="http" http.host=lemmy.website http.target=/api/v3/ws otel.kind="server" request_id=d67d75f7-cd7b-42c1-a8ae-72fbc45340c3 http.status_code=101 otel.status_code="OK"}: lemmy_server::api_routes_websocket: email_not_verified: email_not_verified
lemmy_1     |    0: lemmy_api::local_user::login::perform
lemmy_1     |            with self=Login { username_or_email: Sensitive, password: Sensitive }
lemmy_1     |              at crates/api/src/local_user/login.rs:17
lemmy_1     |    1: lemmy_server::root_span_builder::HTTP request
lemmy_1     |            with http.method=GET http.scheme="http" http.host=lemmy.website http.target=/api/v3/ws otel.kind="server" request_id=d67d75f7-cd7b-42c1-a8ae-72fbc45340c3 http.status_code=101 otel.status_code="OK"
lemmy_1     |              at src/root_span_builder.rs:16

```
- What OS are you trying to install lemmy on?
Ubuntu 22.04
- Any browser console errors?
No

**Comment by k5jat** [comment]
Also, I installed it through ansible. Is there any way I can mark my email address as verified?
Thank you!

**Comment by cereal7802** [comment]
Another user (frorayz) posted in another issue where I described this issue you are having, and provided this link.

https://lemmy.click/comment/973


In that post it explains how to manually modify the database so the admin user email is verified.

My solution was to not enable the email validation, then change the email associated with the admin account. When you change the email it automatically sends a validation email. You can then enable the require validation setting and not get logged out of the admin account.

Essentially there either needs to be a trigger of the email validation when the admin user is created, or it needs to automatically be set as validated when created without the need of an email. I personally like using the email method because it can be used as a signal for functioning email on your instance right at setup time.

**Comment by k5jat** [comment]
I will try your solution. Because not everyone runs an email service when they first start. in my honest opinion, the initial admin email needs to be marked as verified at setup. This is so they can go back and check the "Require email verification" option, it's not an issue. Or, when the option is checked, it triggers a validation email to all accounts with unvalidated emails, which would probably be the better option.

Why this wasn't thought of before is a mystery to me. Thank you!

**Comment by k5jat** [comment]
Ok, changing my email address worked and the site sent the verification email. Luckily, no one's signed up on my instance yet. This is an issue that really should be addressed sooner than later.

**Comment by c0mmando** [comment]
I resolved this issue by modifying the admin user verification boolean in postgres.

**Comment by k5jat** [comment]
> Side note, resetting your password seems to be a backdoor to logging in with an unverified account.

Yeah, that's how I got back in to turn off email verification. I ended up just changing my email address while the email verification was on and I was able to verify that way.

**Comment by kartikynwa** [comment]
@k5jat Did you have to verify your email after that or were you able to log in without verifying?

**Comment by k5jat** [comment]
> @k5jat Did you have to verify your email after that or were you able to log in without verifying?

When I changed my email address and got the verification email, I verified it. No problems since.

**Comment by Racle** [comment]
> Side note, resetting your password seems to be a backdoor to logging in with an unverified account.

Can confirm this. Just logged in to another instance and was wondering why I couldn't login to jerboa. 
After trying to login via another browser instance it said "Email not verified."

Seems like resetting password should be fixed. Or just don't login to lemmy automatically.
